pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -144,7 +144,22 @@ self: super: {
     then dontCheck (overrideCabal super.hakyll (drv: {
       testToolDepends = [];
     }))
-    else super.hakyll;
+    else overrideCabal super.hakyll {
```

Comment:
After fixing the above two problems, can you add a comment here that links to the PR containing these patches and explains why they are necessary?